src/main/java/io/jenkins/plugins/coverage/exception/ConversionException.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.plugins.coverage.exception;
+
+public class ConversionException extends RuntimeException {
```

Comment:
Please avoid using `RuntimeException`s in the production code. It would be better to make it a common exception and define interfaces and error propagation correctly